.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  build_css:
+    runs-on: ubuntu-latest
+    env:
+      source: public/dist/assets
+      destination: assets/website-2021/pr-${{ github.event.number }}/
+    steps:
+      - name: Checkout source Git branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          persist-credentials: false
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Install npm modules
+        run: npm install
+
+      - name: compile assets
+        run: npm run build
+
+      - name: Upload to cdn-dev.w3.org bucket
+        if: success()
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: 'us-east-1'
+        run:
+          aws s3 sync --no-progress ${{ env.source }} s3://cdn-dev.w3.org/${{ env.destination }}
```

Comment:
I don't know if we want to use cdn-dev or cdn.  I think cdn-dev is restricted, and if so it may be difficult to show changes to external contributors.